Edit pass on Monitor Ingestion and Query READMEs and samples (#31626)
Azure/azure-sdk-for-net: sdk/monitor/Azure.Monitor.Query/README.md

This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [cla.microsoft.com][cla].

> When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately. For example, labels and comments. Follow the instructions provided by the bot. You'll only need to sign the CLA once across all repos using our CLA.

Reviewer: ```suggestion
When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately with labels and comments. Follow the instructions provided by the bot. You'll only need to sign the CLA once across all Microsoft repos.
```

Reviewer: nit
